compiler/rustc_typeck/src/check/expr.rs

```diff
@@ -1348,29 +1352,44 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         span: Span,
         remaining_fields: FxHashMap<Ident, (usize, &ty::FieldDef)>,
     ) {
-        let tcx = self.tcx;
```

Comment:
Seemed like a leftover. `tcx` is only used once and so this is unnecessary. Although this is a little bit out of scope, I hope it is somewhat acceptable. (Also see below where I added the `self.`)

Reply:
So, I'm not a maintainer or anything and don't know what the policy is on things like this, but personally I'm a big fan of the Linux kernel's ["one logical change per commit" rule](https://www.kernel.org/doc/html/v4.12/process/submitting-patches.html#separate-your-changes). In this case it would be nice to move this change into it's own commit but no need for a separate PR. Just my 2 cents.

Reply:
Good idea! In the future I will take more care to commit more often and group different changes into different commits.

Reply:
You could still split it up an force-push. I know that @rust-highfive tells contributors to only add new commits but that's very much a judgement call and there's been some talk about softening the message.

At that opportunity you could also change "Fix tests" to "Bless tests", as that's the accepted lingo for what that commit does. After all, the tests weren't broken, the just didn't pass anymore.

Reply:
I'm not yet very experienced with `git` and I feel like I will mess something up. But I think if this Pull Request is merged, isn't it going to be squashed into one commit anyway?

Reply:
I see. Depending on your time and motivation I'd encourage you to take the search engine of your choice, make a playground branch or even a playground repository, and trying it out. It's really not hard and a valuable skill to have.

There may be projects that automatically squash PRs, but Rust isn't one of them.

Reply:
Tiny cleanup in the same code seems fine.